packages/perseus/src/widgets/radio/choice-icon/choice-icon.jsx

```diff
@@ -31,57 +33,200 @@ type ChoiceIconProps = {|
     transparentBackground?: boolean,
 
     primaryProductColor?: string,
-|};
+};
+
+function ChoiceInner(props: {
+    pos: number,
+    showCorrectness: boolean,
+    correct: ?boolean,
+}) {
+    const {pos, showCorrectness, correct} = props;
+    const letter = getChoiceLetter(pos);
+
+    if (!showCorrectness) {
+        return letter;
+    }
+    if (correct) {
+        return (
+            <InlineIcon
+                {...iconCheck}
+                style={{
+                    position: "relative",
+                    top: -1,
+                }}
+            />
+        );
+    }
+    return <InlineIcon {...iconMinus} />;
+}
+
+// Handle dynamic styling of the multiple choice icon. Most
+// MC icon styles are constant, but we do allow the caller
+// to specify the selected color, and thus must control styles
```

Comment:
Might be beyond the scope of this, but I'm curious if this is still true. I feel 10% sure that we don't let radio colors get overwritten in the "library" radio and SAT is gone. NAN, but I wonder if we should followup on whether we still need this logic.

Reply:
I'll review.  I suspect it _was_ just SAT customization that needed this.

Reply:
Ok, good call! I can remove the `styling` prop (which has the `primaryProductColor`) from APIOptions entirely!!! :D